.travis.yml

```diff
@@ -24,7 +24,7 @@ env:
     - LIBSODIUM_DIR=${TRAVIS_BUILD_DIR}/libsodium
     - AMCL_TAG=4.7.0
     - AMCL_DIR=${TRAVIS_BUILD_DIR}/milagro-crypto-c/install
-    - XAPTUM_TPM_TAG=v0.5.6
```

Comment:
Oh, I just now understand this: the updates to xaptum-tpm that this requires haven't been released yet. I'm sorry.

I'll cut a new release, 0.5.7, so we can pin to that one, rather than to master.